### ISSUE TITLE
Release

### ISSUE THREAD

**Issue by greimel** [baggepinnen/LiterateWeave.jl#3]
Do you plan to realease this package? I would like to use it to automatically generate documentation. If it's released it saves me one line in the CI script ;-)

**Comment by baggepinnen** [comment]
Yeah sure! @fredrikekre would you have any objections to the release of this package considering it's name and scope? (It's a glue package between Literate.jl and Weave.jl.)

**Comment by fredrikekre** [comment]
> @fredrikekre would you have any objections to the release of this package considering it's name and scope?

No, not at all. I am curious about this from the README though:

> The Juno cell-separator ## confuses Literate and produces an invalid character which causes Weave to fail.

**Comment by baggepinnen** [comment]
Sweet! 
Ah, great if that's no longer an issue =)

**Comment by fredrikekre** [comment]
I mean, I don't know. What was the issue?

**Comment by baggepinnen** [comment]
It used to produce an invisible unvalid character or something like that. But I just tested and now it treats `##` like an inline comment, so I should update the readme to reflect that.

**Comment by baggepinnen** [comment]
TODO: Some tests would be good to have before registering this package
